src/config.rs

```diff
@@ -55,16 +48,18 @@ pub struct TermvizConfig {
     pub marker_topics: Vec<ListenerConfig>,
     pub image_topics: Vec<ListenerConfig>,
     pub marker_array_topics: Vec<ListenerConfig>,
+    pub send_pose_topic: String,
     pub target_framerate: i64,
     pub axis_length: f64,
     pub visible_area: Vec<f64>, //Borders of map from center in Meter
     pub zoom_factor: f64,
-    pub teleop_key_mapping: TeleopConfig,
+    pub key_mapping: HashMap<String, String>,
+    pub teleop: TeleopConfig,
 }
 
 impl Default for TermvizConfig {
     fn default() -> Self {
-        let conf = TermvizConfig {
+        TermvizConfig {
```

Comment:
The config is not backwards compatible and you suggest more not backwards compatible changes. I propose to separate that to a different PR, that one will probably spawn a lot of discussions

Reply:
How do I maintain compatibility? Should I duplicate all the settings and write a compatibility layer in config.rs? The unified input system is one of the main new features of this PR...

Reply:
We decided not to care about backward compatibility at this point in time